PowerArubaSW/Public/LACP.ps1

```diff
@@ -0,0 +1,162 @@
+#
+# Copyright 2018, Alexis La Goutte <alexis.lagoutte at gmail dot com>
+# Copyright 2018, C�dric Moreau <moreaucedric0 at gmail dot com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+function Get-ArubaSWLACP {
+
+    <#
+        .SYNOPSIS
+        Get LACP information about the ports in a lacp configuration on ArubaOS Switch.
+
+        .DESCRIPTION
+        Get the list of all ports in a lacp configuration.
+
+        .EXAMPLE
+        Get-ArubaSWLACP
+        This function give you the list of ports in a lacp configuration with the link aggregation interface and the name of ports in this interface.
+    #>
+
+    Begin {
+    }
+
+    Process {
+
+        $url = "rest/v4/lacp/port"
+
+        $response = invoke-ArubaSWWebRequest -method "GET" -url $url
+
+        $run = ($response | convertfrom-json).lacp_element
+
+        foreach ($run in $run)
+        {
+            $status = [pscustomobject]@{
+            LocalPort  = $run.port_id
+            TrunkGroup = $run.trunk_group
+            }
+
+            $status
+        }
+    }
+
+    End {
+    }
+}
+
+function Add-ArubaSWLACP {
+
+    <#
+        .SYNOPSIS
+        Add ports into a lacp configuration on ArubaOS Switch.
+
+        .DESCRIPTION
+        Add ports in a lacp interface.
+
+        .EXAMPLE
+        Add-ArubaSWLACP <port> <interface>
+        This function allow you to add port in a lacp configuration with the name of the lacp interface and the name of the port.
+
+        .EXAMPLE
+        Add-ArubaSWLACP -port [X] -interface [trkY]
+        This function allow you to add port in a lacp interface with the parameter "interface" for the name of the lacp interface (examples : trk1, trk2 ....)
+        and "port" for the name of the port that you want to add to this lacp interface.
+
+        .EXAMPLE
+        If you want to configure ports 3 and 5 in trunk group 6 :
```

Comment:
Always PS cmdlet before test
and if there is multiple line, for second line use  PS C:\>